scripts/tools/convert_limit_mm_to_usteps.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+# Constants
+CONFIG_SCREW_PITCH_MM_XY = 1  # Screw pitch for X and Y axes in mm
+CONFIG_SCREW_PITCH_MM_Z = 0.012 * 25.4  # Screw pitch for Z axis in mm (0.012 inches to mm)
+STEPS_PER_REVOLUTION = 200 * 8  # Full steps per revolution times microstepping factor
+
+# Edge positions in mm
+edge_positions_mm = [
```

Comment:
These should not be hardcoded, can we put them in a file which generated during a calibration process?

Reply:
This is a very old file. The current version of 'mm_to_usteps' is here: [https://github.com/cccoolll/squid-control/blob/156f9e188df9007d5fd3d7ea42004a3bfdffbf4b/squid_control/control/utils/generate_software_limit.py#L19](url)

So we should remove this file.